tartufo/scanner.py

```diff
@@ -591,40 +591,82 @@ def scan_entropy(
         :param chunk: The chunk of data to be scanned
         """
 
-        for line in chunk.contents.split("\n"):
-            # The lines are in diff format and so contain a prefixed +/-.
-            # We are ignoring the first element in each line so the Tartufo matched_string
-            # reports the actual string that generates the entropy finding.
-            for word in line[1:].split():
+        for line in (x for x in chunk.contents.split("\n") if x):
+            # If the chunk is diff output, the first character of each line is
+            # generated metadata ("+", "-", etc.) that is not part of actual
+            # repository content, and it should be ignored.
+            extra_char: Optional[str]
+            if chunk.is_diff:
+                extra_char = line[0]
+                analyze = line[1:]
+            else:
+                extra_char = None
+                analyze = line
+            for word in analyze.split():
                 for string in util.find_strings_by_regex(word, BASE64_REGEX):
                     yield from self.evaluate_entropy_string(
-                        chunk, line, string, self.b64_entropy_limit
+                        chunk, line, string, self.b64_entropy_limit, extra_char
```

Comment:
Shouldn't this be `anazlye`? And same for the next one?
```suggestion
                        chunk, analyze, string, self.b64_entropy_limit, extra_char
```